src/main/java/io/gravitee/reporter/api/v4/metric/native_kafka/NativeEventMetrics.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.reporter.api.v4.metric.native_kafka;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.gravitee.reporter.api.AbstractReportable;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Getter
+@Setter
+public class NativeEventMetrics extends AbstractReportable {
+
+    @JsonProperty("topic")
+    private String topicName;
+
+    @JsonProperty("partition")
+    private String partition;
+
+    @JsonProperty("plan-id")
+    private String planId;
+
+    @JsonProperty("api-id")
+    private String apiId;
+
+    @JsonProperty("app-id")
+    private String applicationId;
+
+    @JsonProperty("gw-id")
+    private String gatewayId;
+
+    @JsonProperty("env-id")
+    private String environmentId;
+
+    @JsonProperty("org-id")
+    private String organizationId;
+
+    @JsonProperty("downstream_publish_messages_total")
+    private Number downstreamPublishMessagesTotal;
+
+    @JsonProperty("downstream_publish_message_bytes")
+    private Number downstreamPublishMessageBytes;
+
+    @JsonProperty("upstream_publish_messages_total")
+    private Number upstreamPublishMessagesTotal;
+
+    @JsonProperty("upstream_publish_message_bytes")
+    private Number upstreamPublishMessageBytes;
+
+    @JsonProperty("downstream_subscribe_messages_total")
+    private Number downstreamSubscribeMessagesTotal;
+
+    @JsonProperty("downstream_subscribe_message_bytes")
+    private Number downstreamSubscribeMessageBytes;
+
+    @JsonProperty("upstream_subscribe_messages_total")
+    private Number upstreamSubscribeMessagesTotal;
+
+    @JsonProperty("upstream_subscribe_message_bytes")
+    private Number upstreamSubscribeMessageBytes;
+
+    public NativeEventMetrics(
+        @NonNull String topicName,
+        String partition,
```

Comment:
IMHO this one can be removed and we will see in a feature improvement to add it with `downstream_subscribe_messages_lag`. WDYT ?